### ISSUE TITLE
BXT216 Bowflex Treadmill Does not work in myWhoosh but works in Zwift[BUG]

### ISSUE THREAD

**Issue by hpcolus** [cagnulein/qdomyos-zwift#2204]
The TM BXT216 works in Zwift (avatar runs) but in myWhoosh the avatar does not move at all.  I followed your suggestion to rename the Android phone to three letters(from 'Pixel 6' to 'PhP'). Still that does not work.
**Desktop
-OS: Windows 11
--Version 22H2
**Smartphone
-Device: Pixel 6
--OS: Android
--Version: 14

**Comment by cagnulein** [comment]
On zwift which device do you pair? Wahoo?

**Comment by hpcolus** [comment]
On Zwift, I see only the name of my phone 'bluetooth' wich in my case is 'PhP'
![image_2024-03-10_111839143](https://github.com/cagnulein/qdomyos-zwift/assets/104551107/1533e453-4740-4ea4-b313-20efd6d07fce)

**Comment by cagnulein** [comment]
ok so that's strange, i guess it's a mywhoosh bug. did you contact them? mywhoosh is strongly in beta for run

**Comment by hpcolus** [comment]
Yes, I did contact MW.  I got the impression that not many (or any) treadmill has been tested for support.  Here is their response.  It will take some time before we see this functionality being implemented.   

`MyWhoosh has been tested and verified to work with specific devices, including the Assault, Polar running pod, and Under Armour smart shoes. Unfortunately, compatibility with other running equipment, such as the QZ application and your treadmill, may not be guaranteed.`
